data/strings/types_strings.txt

```diff
@@ -11849,7 +11851,7 @@
     bg = Дворец
     cs = Palác
     da = Palads
-    de = Palais
+    de = Palast
```

Comment:
I personally prefer 'Palais' and looking at overpass I see that almost all castles in Germany and Austria are called Palais x rather than Palast. What do you think @Mr-Mime ?

Reply:
Looks like in Germany most castle:type=palaces are called 'Schloss'. In Austria however 'Palais' is common.

Reply:
@map-per yes, you can start adding de-AT, we will later support it.

Reply:
Sure, but there's no official german translation for Buckingham Palace. ;)
I mean, just look at the wikipedia list for [Germany](https://de.wikipedia.org/wiki/Kategorie:Palast_in_Deutschland) and [Austria](https://de.wikipedia.org/wiki/Kategorie:Palais_in_%C3%96sterreich). There's not a single palace called 'Palast'. That's why it would sound unnatural to me to call any of them Palast.

Reply:
But we can let the majority decide, we don't need to differentiate between de and de-AT for that.

Reply:
> How would you call the Buckingham Palace in German? 'Palast' or 'Palais'?

What I wanted to say is: The translation will be used in the entire world. Would you also call a palace outsiede of Germany, Austria and Switzerland 'Palais'?